lib/ueberauth/strategy/facebook.ex

```diff
@@ -193,10 +217,12 @@ defmodule Ueberauth.Strategy.Facebook do
 
   defp option(conn, key) do
     default = Keyword.get(default_options(), key)
-
-    conn
-    |> options
-    |> Keyword.get(key, default)
+    case options(conn) do
+      nil ->
+        Keyword.get([], key, default)
```

Comment:
That not make sense. The default value will always be used because Keyword is empty there.